nss-day-cohort-34/workforce-management-awkward-conversations-and-gravy
# Bj employeedetails

# Description

When user clicks on details on the employees page then that employee's full name, department, currently assigned computer, and list of training programs user is signed up for should be displayed.

Related to Issue #  3


# Testing Instructions

Click Employees up top then click on details next to employees to make sure data is displayed correctly.

# Checklist:
